data processing (#6)
UBC-MDS/Cost_of_living_py: Proposal.md

# Section 3: Research questions and usage scenarios

Our project answers the broad research question of: “What is the average
cost of living for a single fresh graduate in different cities in the
world?”. Below we have provided a usage scenario of our product by a
member of our target audience:

Bella is a fresh graduate in Canada and her current marital status is
single. She wants to seek for a job and settle down in some major cities

> in the world, but she concerns about the high living cost in big cities,

Reviewer: ```suggestion
in the world, but she has concerns about the high living cost in big cities,
```

including housing, grocery, transportation, entertainment, shopping and

> so on. In order to help her get a general idea of how much the living

Reviewer: ```suggestion
others. In order to help her get a general idea of how much the living
```
